keyboards/handwired/wwa/soyuz/keymaps/default/rules.mk

```
@@ -0,0 +1,8 @@
+CONSUMER_ENABLE = no
+VIA_ENABLE = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
+MIDI_ENABLE = yes
+AUDIO_ENABLE = no
+ANALOG_ENABLE = yes
+ANALOG_DRIVER_REQUIRED = yes
```

```suggestion
ENCODER_MAP_ENABLE = yes
```
Most of these should not be present in a `default` keymap, per the PR Checklist, and if `ENCODER_ENABLE=yes` is already present in the keyboard level setting then it should not be duplicated here

`VIA_ENABLE` should only be present in the `via` keymap - that would need to be a separate PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository